asteroidsyncserviced/platforms/ubuntutouch/platform.cpp

```diff
@@ -62,7 +62,7 @@ uint Platform::Notify(const QString &app_name, uint replaces_id, const QString &
             qDebug() << __func__ << "Have a phone call notification. Ignoring it..." << app_name << app_icon;
         } else if(m_notificationServiceReady) {
             unsigned int randId = rand() % 100000 + 1;
-            m_notificationService->insertNotification("", randId, encodeAppName(app_name), encodeIcon(app_name), summary, body, m_notificationService->getVibration());
+            m_notificationService->insertNotification("", randId, encodeAppName(app_name), encodeIcon(app_name), summary, body, m_notificationService->strToVibrate("normal"));
```

Comment:
If you call strToVibrate here only to convert it back to a string right away (in libasteroid). How about just forwarding a string from the beginning instead ? We could remove all the enum/string conversion logic from libasteroid too

Reply:
There are only four valid values and no way to indicate that via D-Bus.  An alternative would be to use an int instead, which I think would be neater.  Which do you prefer?

Reply:
Yes, they're all using QString now